### DISCUSSION TITLE
tab completion is crucial

### DISCUSSION THREAD

**Discussion by zvnkk** [WLJSTeam/wljs-notebook#301] [Ideas]
Hello, 
You are doing great work.
Could you add tab completion in the edior, it could be not enabled by default if you don't want to, however there should be something that you could toggle to switch on and off.

**Comment by JerryI** [comment]
Hello!
Actually why not, we need to check the settings of code mirror 6 editor

**Comment by JerryI** [comment]
Added in [commit](https://github.com/JerryI/wljs-editor/commit/f0c0bff4bf0a7116987bd46e9a203929be7eb6d0)

To fetch a new version without any delays:
1. open settings, click `check updates`
2. restart an app
3. clear cache (from the menu -> window reload or force reload)

**Comment by zvnkk** [comment]
Tab completion works now 🔥 🔥 🔥, many thanks.
